go/registry/api/runtime.go

```diff
@@ -231,10 +231,43 @@ type RuntimeGenesis struct {
 	StateRoot hash.Hash `json:"state_root"`
 
 	// State is the state identified by the StateRoot. It may be empty iff
-	// the StorageReceipt is not invalid or StateRoot is an empty hash.
+	// all StorageReceipts are valid or StateRoot is an empty hash or if used
+	// in network genesis (e.g. during Tendermint InitChain).
 	State storage.WriteLog `json:"state"`
 
-	// StorageReceipt is the storage receipt for the state root. It may be
-	// invalid iff the State is non-empty or StateRoot is an empty hash.
-	StorageReceipt signature.Signature `json:"storage_receipt"`
+	// StorageReceipts are the storage receipts for the state root. The list
+	// may be empty or a signature in the list invalid iff the State is non-
+	// empty or StateRoot is an empty hash or if used in network genesis
+	// (e.g. during Tendermint InitChain).
+	StorageReceipts []signature.Signature `json:"storage_receipts"`
+
+	// Round is the runtime round in the genesis.
+	Round uint64 `json:"round"`
+}
+
+// SanityCheck does basic sanity checking of RuntimeGenesis.
+// isGenesis is true, if it is called during Tendermint's InitChain.
```

Comment:
The comment shouldn't involve Tendermint, say "during consensus chain init" or "network genesis" or something (unify with docs in field descriptions above).